lib/internal/child_process.js

```diff
@@ -577,7 +577,7 @@ function setupChannel(target, channel) {
         handle: null,
         message: message,
       });
-      return;
+      return this._handleQueue.length < (65536 * 2);
```

Comment:
128k queued messages is perhaps a bit much.  Maybe return false when the length is > 1?